lib/src/picture_stream.dart

```diff
@@ -237,7 +238,7 @@ abstract class PictureStreamCompleter extends Diagnosticable {
         List<_PictureListenerPair>.from(_listeners);
     for (_PictureListenerPair listenerPair in localListeners) {
       try {
-        listenerPair.listener(picture, false);
+        listenerPair.listener(picture, false, _error);
```

Comment:
I think if this throws the error builder won't get called, right?

Reply:
```dart
void _handleImageChanged(PictureInfo imageInfo, bool synchronousCall, bool error) {
    setState(() {
      _picture = imageInfo;
	  _error = error;
    });
  }
```

It's catch in the function above (in the `_SvgPictureState` class).
I don't know where the error builder should be call apart in `_SvgPictureState`.

Reply:
If you look at line 243, there's a check for an error listener.  We should be bubbling that back up as well.